client/utils/handle-logout.js

```diff
@@ -0,0 +1,24 @@
+import {toast} from "react-toastify";
+import {t} from "ttag";
+import {initialState} from "../reducers/organization";
+
+const redirectToStatus = (setUserData, userData, orgSlug) => {
+  setUserData({...userData, mustLogout: true});
+  window.location.assign(`${window.location.origin}/${orgSlug}/status`);
+};
+
+const handleLogout = (logout, cookies, orgSlug, setUserData, userData) => {
+  if (
+    userData.is_active === true &&
+    (userData.method === "" ||
+      (userData.method === "mobile_phone" && userData.is_verified === true) ||
+      (userData.method === "bank_card" && userData.is_verified === false))
+  ) {
```

Comment:
it's not super clear what happens here, I think we should add some comments.

I think the user should always be logged out from the captive portal if is_verified is true or if the method used is bank_card or unspecified (regardless of is_verified).